cypress/cypress/component/Formatted/NumericFormat.cy.tsx

```diff
@@ -48,6 +49,39 @@ it("is disabled", () => {
   cy.get(`input[name=${name}]`).should("be.disabled");
 });
 
+it("auto mark on focus", () => {
+  const name = faker.random.word();
+  const randomNumber = faker.datatype.number({
+    min: 10000000,
+  });
+
+  cy.mount(
+    <Form
+      defaultValues={{ [name]: randomNumber }}
+      onSubmit={() => {
+        // Do nothing
+      }}
+    >
+      <FormattedInput name={name} label={name} numericFormat={numericFormat} markAllOnFocus />
+    </Form>,
+  );
+
+  cy.contains("label", name).click();
+  let selectedText: string | undefined;
+  cy.get(`input[id=${name}]`)
+    .then((input) => {
+      selectedText = getSelectedTextFromInputField(input);
```

Comment:
detail: but i think you could wrap the result to use it in the next then. 

Would mean you don't have to have a `let` variable which we usually try to avoid.

Reply:
i think now its a little bit better 😄

Reply:
thx!